pyirf/cut_optimization.py

```diff
@@ -86,6 +86,8 @@ def optimize_cuts(
     )
 
     with tqdm(total=n_cuts, disable=not progress) as bar:
+        all_the_gh_cuts = []
```

Comment:
maybe call this `gh_cut_grid`

`all_the_...` was an example I wrote in chat to clarify the point... not really a good variable name choice I have to admit, same for the theta cuts